src/object/schema_manager.c

```diff
@@ -15025,7 +15025,20 @@ sm_is_possible_to_recreate_constraint (MOP class_mop, const SM_CLASS * const cla
 
   if (class_->users != NULL)
     {
-      return false;
+      if (class_->partition != NULL)
```

Comment:
Not related to this line.

Can you explain why other "constraint->type"s (eg. SM_CONSTRAINT_PRIMARY_KEY) are impossible to recreate?
- I want to know why it is impossible conceptually.
- and I want to know the exact criterion between possible and impossible.